### ISSUE TITLE
Migrate from React Context API to Easy Peasy for managing local state

### ISSUE THREAD

**Issue by muco-rolle** [burundiarxiv/burundiarxiv-web-client#27]
React Context API is good for small applications but it is unopinionated which can be hard to understand for other contributors as you can structure it the way you want. so we gonna migrate to easy-peasy which an abstraction of Redux, providing a reimagined API that focuses on developer experience.

https://easy-peasy.now.sh/

**Comment by lionel-k** [comment]
Can you please in this ticket try to make the code simpler ? As we discussed, we introduced complex architecture for a project which is still small. What do you think ? 🤔

**Comment by muco-rolle** [comment]
Yeah, sure I'll try to see how we can do without a state management system.
